lib/tasks/tiny_mce_asset.rake

```diff
@@ -0,0 +1,10 @@
+namespace :tiny_mce_asset do
+  desc 'Remove obsolete images that were created on new steps or '\
+       'results and the step/result didn\'t get saved.'
+  task remote_obsolete_images: :environment do
```

Comment:
:<. I know, I just said the naming of the function isn't OK. Just rename it from `remote_obsolete_images` to `remove_obsolete_images`. Do you have dyslexia?: https://www.understood.org/en/learning-attention-issues/child-learning-disabilities/dyslexia/understanding-dyslexia.